README.md

```diff
@@ -29,8 +29,9 @@ complete the following tasks:
   [database](https://app.bugsplat.com/v2/settings/company/databases)
   for your application
 - Check out the
-  [live demo](https://www.bugsplat.com/platforms/react/my-react-crasher)
-  of BugSplat’s error reporting for React
+[live demo](https://www.bugsplat.com/platforms/react/my-react-crasher)
+of BugSplat’s error reporting for React.
+  - source: [/examples/my-react-crasher](/examples/my-react-crasher).
```

Comment:
Please update url for live demo to https://bugsplat-git.github.io/my-react-crasher/